docs/operator-guides/configuring-airbyte.md

```diff
@@ -96,13 +96,14 @@ The following variables are relevant to both Docker and Kubernetes.
 ### Kubernetes-Only
 #### Jobs
 1. `JOB_KUBE_TOLERATIONS` - Define one or more Job pod tolerations. Tolerations are separated by ';'. Each toleration contains k=v pairs mentioning some/all of key, effect, operator and value and separated by `,`.
-2. `JOB_KUBE_NODE_SELECTORS` - Define one or more Job pod node selectors. Each kv-pair is separated by a `,`.
-3. `JOB_KUBE_MAIN_CONTAINER_IMAGE_PULL_POLICY` - Define the Job pod connector image pull policy.
-4. `JOB_KUBE_MAIN_CONTAINER_IMAGE_PULL_SECRET` - Define the Job pod connector image pull secret. Useful when hosting private images.
-5. `JOB_KUBE_SOCAT_IMAGE` - Define the Job pod socat image.
-6. `JOB_KUBE_BUSYBOX_IMAGE` - Define the Job pod busybox image.
-7. `JOB_KUBE_CURL_IMAGE` - Define the Job pod curl image pull.
-8. `JOB_KUBE_NAMESPACE` - Define the Kubernetes namespace Job pods are created in.
+2. `JOB_KUBE_NODE_SELECTORS` - Define one or more Job pod node selectors. Each k=v pair is separated by a `,`. For example: `key1=value1,key2=value2`
+3. `JOB_KUBE_ANNOTATIONS` - Define one or more Job pod annotations. Each k=v pair is separated by a `,`. For example: `key1=value1,key2=value2`
```

Comment:
can we update the comment here to match my earlier comment on reflecting that this the default annotation fallback and the annotation configuration for sync jobs.